src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { findUser } from '../../api/api';
+import { UserType } from '../../types/UserType';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+interface Props {
+  userId: number,
+  deleteUser: () => void
+}
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = ({ userId, deleteUser }) => {
+
+  const [user, setUser] = useState<UserType | null>(null);
+
+  const getUser = async () => {
+    setUser(await findUser(userId));
+  };
```

Comment:
you have to handle errors, if findUser returns an error you should catch error here